### DISCUSSION TITLE
Change path of config file

### DISCUSSION THREAD

**Discussion by alverman** [sumatrapdfreader/sumatrapdf#3483] [Q&A]
I'm still here .
I thought I read that you could assign a personal path to the configuration file.
Am I wrong ?
Maybe I am wrong because I can't find the entry .

Thanks, Alberto

**Comment by GitHubRulesOK** [comment]
on calling sumatrapdf.exe add `-appdata c:\blah\blah\folder`
 but not needed for portable which will use same folder as exe  by default
